app/input_objects/pages/conditions_input.rb

```diff
@@ -65,6 +65,16 @@ def assign_skip_to_end
     end
   end
 
+  def next_page_number
+    if page.has_next_page?
+      target_page = FormRepository.pages(form).find { it.id == page.next_page }
+      question_number = target_page.position
+      return question_number
+    end
+
+    I18n.t("page_route_card.check_your_answers")
+  end
```

Comment:
In what scenario would this return `check your answers`?

The reason I ask is because if it did, then `continue_to_question_number` could be `check your answers`, and then the content for `pages.conditions.new.body_html` would end with "question check your answers" 🤔

Reply:
thats a good point!

As it's only shown on the new screen, we can be sure it's never check your answers because there must always be a question between the start of the route and the end of the form.

I've removed it and made a note to check with designers what should appear on the edit screen.